doc/releases/changelog-dev.md

```diff
@@ -389,6 +389,9 @@
 * `qml.grouping.is_pauli_word` now returns `False` for operators that don't inherit from `qml.Observable`, instead of raising an error.
   [(#3039)](https://github.com/PennyLaneAI/pennylane/pull/3039)
 
+* `qml.grouping.utils.is_commuting` is removed, and its Pauli word logic is now part of `qml.is_commuting`.
+  [(#3033)](https://github.com/PennyLaneAI/pennylane/pull/3033)
+
```

Comment:
Might be worth sticking this in "Breaking changes". I don't think most people will care, but better safe than sorry.

Reply:
agreed! changed.